src/components/NavForm.jsx

```diff
@@ -225,6 +225,16 @@ export default styled(withAmplitudeHooks(NavForm))({
   "& .checkboks[required] + .skjemaelement__label::after": {
     content: '""',
   },
+  "& .skjemaelement__rad": {
```

Comment:
Hva med å ha en fil med common styling (js objekter) som vi kan "extende" inn i stylingen for hver component.

Reply:
Har tenkt tanken selv. Hadde egentlig ikke tenkt å ta det som en del av denne oppgaven, men godt med alt som er gjort.